<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormECG</class>
 <widget class="QWidget" name="FormECG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="ecg1LeadSetComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>Ⅰ</string>
   </property>
   <item>
    <property name="text">
     <string>Ⅰ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ⅱ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ⅲ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ecg1LeadSetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ECG1导联设置</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ecg1GainSetComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>X0.25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X0.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ecg1GainSetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ECG1增益设置</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ecg2LeadSetComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>Ⅰ</string>
   </property>
   <item>
    <property name="text">
     <string>Ⅰ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ⅱ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ⅲ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AVF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ecg2LeadSetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ECG2导联设置</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ecg2GainSetComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>X0.25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X0.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>X2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ecg2GainSetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ECG2增益设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
